tests/components/spc/conftest.py

```diff
@@ -8,19 +8,26 @@
 
 
 @pytest.fixture
-def mock_client() -> Generator[AsyncMock]:
+def mock_clients() -> Generator[tuple[AsyncMock, AsyncMock]]:
     """Mock the SPC client."""
+    with (
+        patch("homeassistant.components.spc.SpcWebGateway", autospec=True) as init_mock,
+        patch(
+            "homeassistant.components.spc.config_flow.SpcWebGateway", autospec=True
+        ) as config_flow_mock,
+    ):
```

Comment:
```suggestion
    with (
        patch("homeassistant.components.spc.SpcWebGateway", autospec=True) as init_mock,
        patch(
            "homeassistant.components.spc.config_flow.SpcWebGateway", new=init_mock
        ),
    ):
```

This way you don't need the for loop